contracts/zero-ex/contracts/test/foundry/forked/SwapNativeToERC20RFQTv2.sol

```diff
@@ -0,0 +1,171 @@
+pragma solidity ^0.6;
+
+pragma experimental ABIEncoderV2;
+
+import "../utils/ForkUtils.sol";
+import "../utils/TestUtils.sol";
+import "src/IZeroEx.sol";
+import "@0x/contracts-erc20/contracts/src/v06/IEtherTokenV06.sol";
+import "src/features/TransformERC20Feature.sol";
+import "src/external/TransformerDeployer.sol";
+import "src/transformers/WethTransformer.sol";
+import "src/transformers/FillQuoteTransformer.sol";
+import "src/transformers/bridges/BridgeProtocols.sol";
+import "src/features/OtcOrdersFeature.sol";
+
+contract SwapNativeToERC20RFQTv2 is Test, ForkUtils, TestUtils {
```

Comment:
This seems like a test that could also have a `local` version. I think @patrick-dowell could use some guidance in setting this up, and potentially to set up some "mocked" AMMs to develop against locally.